### ISSUE TITLE
MotionMark test

### ISSUE THREAD

**Issue by roman6626** [MrAlex94/Waterfox#1248]
Why in the test MotionMark 1.1 the results of Waterfox are much less than those of Basilisk and Pale Moon? Pale Moon - 80,48 Basilisk - 66,98 Waterfox - 36,67 Very small values for: Multiply, Canvas Arcs, Canvas Lines. Repeated many times. The results are the same. It struck me. Why is that?
All the while I thought that Waterfox is a more media browser and should work faster than other legacy-browsers.

**Comment by grahamperrin** [comment]
> Waterfox

Waterfox Classic 2019.10 or Waterfox Current 2019.10?

**Comment by roman6626** [comment]
It's only about Waterfox Classic 2019.10!

**Comment by grahamperrin** [comment]
[MotionMark 1.1 | WebKit](https://webkit.org/blog/8434/motionmark-1-1/) ▶ https://browserbench.org/MotionMark1.1/

For Waterfox Classic the shortest answers (to close this issue) are probably: 

- compare with with Firefox 56.0.2
- be familiar with Mozilla's Project Quantum. 

Historically: Waterfox 56.0 was based on Firefox 56.0.2. 

Most recently: Waterfox Classic 2019.10 is the successor to Waterfox Classic 56.2.14, is still essentially 56-based. 

### Recommended reading 

[Waterfox 55 Release (Windows, Mac, Linux and Android) | Waterfox Web Browser](https://www.waterfox.net/blog/waterfox-55.0-release-download)

[Waterfox 56 Release | Waterfox Web Browser](https://www.waterfox.net/blog/waterfox-56.0-release-download/)

[Quantum Up Close: What is a browser engine? - Mozilla Hacks - the Web developer blog](https://hacks.mozilla.org/2017/05/quantum-up-close-what-is-a-browser-engine/) referred from [A good ELI5 guide for Quantum! … : firefox](https://www.reddit.com/r/firefox/comments/6ls7s2/a_good_eli5_guide_for_quantum_apologies_if/)

[Inside a super fast CSS engine: Quantum CSS (aka Stylo) - Mozilla Hacks - the Web developer blog](https://hacks.mozilla.org/2017/08/inside-a-super-fast-css-engine-quantum-css-aka-stylo/) referred from #332

**Comment by roman6626** [comment]
But Pale Moon and the Basilisk are based on even earlier versions of Mozilla.
I am now talking about the comparison of these browsers, and not with Quantum.
For some reason, I always thought that Waterfox should work faster in a modern web than Pale Moon and Basilisk.

**Comment by roman6626** [comment]
Is there anything that distinguishes Waterfox for the better from Basilisk?

**Comment by grahamperrin** [comment]
> Is there anything that distinguishes Waterfox for the better from Basilisk?

Yes, but such things are unrelated to MotionMark.

**Comment by roman6626** [comment]
Of course not connected. But this test shows the performance of the graphics engine. And this for some reason is not for the better at Waterfox.

**Comment by grahamperrin** [comment]
For me on FreeBSD-CURRENT: 

- [Basilisk was never an option](https://www.freshports.org/search.php?stype=name&method=match&query=Basilisk&num=100&deleted=includedeleted&orderby=category&orderbyupdown=asc&search=Search&format=html&branch=head), and I don't expect a port
- New Moon a.k.a. Pale Moon is no longer available (port maintenance issues). 

### Food for thought

![2019-11-03 18:45:05 MotionMark 1 1, Waterfox Classic 2019 10](https://user-images.githubusercontent.com/192271/68105657-209b8e80-fed7-11e9-9342-7788808736d1.png)

![2019-11-03 19:10:49 MotionMark 1 1, Falkon 3 1 0](https://user-images.githubusercontent.com/192271/68105680-34df8b80-fed7-11e9-8638-3dbdafc62453.png)

![2019-11-03 19:21:49 MotionMark 1 1, Firefox 70 0 1](https://user-images.githubusercontent.com/192271/68105690-3ad56c80-fed7-11e9-81d3-cbc0c7a150f7.png)

![2019-11-03 19:37:56 MotionMark 1 1, Chromium 76 0 3809 132](https://user-images.githubusercontent.com/192271/68105674-2e511400-fed7-11e9-9175-ae925f63b7e3.png)

Those results are not intended to be accurate – I didn't bother with full screen, some memory was swapped to disk, and so on – but I imagine that they're unusual. 

YMMV. Vary massively :-)

**Comment by roman6626** [comment]
I just use only Windows. :-)

**Comment by roman6626** [comment]
Why does Waterfox have 1 position in this test (Multiply) is always 1?

**Comment by MrAlex94** [comment]
Do you have e10s and hardware acceleration enabled?
https://support.mozilla.org/en-US/kb/performance-settings#firefox:win10:fx60

**Comment by roman6626** [comment]
Hey! Tried and so and so. The result is not affected. By the way, Pale Moon and Basilisk do not have multiprocessing. But they always have this value much greater than 0.

**Comment by grahamperrin** [comment]
> Why does Waterfox have 1 position in this test (Multiply) is always 1?

I don't know, but it's not always so. Please see below, where the test result was 72.95. 

>> Pale Moon - 80,48 Basilisk - 66,98 Waterfox - 36,67 

Below: 

- Pale Moon 70.02
- Waterfox Classic 82.91

![image](https://user-images.githubusercontent.com/192271/68803559-e610bf00-0657-11ea-9e5f-7cc83337d3db.png)

**Comment by roman6626** [comment]
Does it depend on the inclusion of multiprocessing in Waterfox or not? There is no multiprocessing at Pale Moon.
On different machines, the results are always the same. Is Waterfox always smaller? What could be the reason?

**Comment by roman6626** [comment]
Checked a few more times. It does not depend on multiprocessing. As before, the value of Multiply = 1! Why?

**Comment by roman6626** [comment]
I can guess, maybe the Pale Moon and the Basilisk are better optimized for older cars? 
(Core i3 M370, 2,40 GHz, 8 Gb)
Could this be the reason?

**Comment by grahamperrin** [comment]
Mozilla bug [1553575 - MotionMark 1.1 - test "Multiply": scores with Firefox are inconsistent](https://bugzilla.mozilla.org/show_bug.cgi?id=1553575) refers to: 

- https://browserbench.org/MotionMark1.1/developer.html?test-interval=30&display=minimal&tiles=big&controller=ramp&frame-rate=50&kalman-process-error=1&kalman-measurement-error=4&time-measurement=performance&suite-name=MotionMark&test-name=Multiply

![image](https://user-images.githubusercontent.com/192271/69009097-2f0f8e80-094a-11ea-9523-c8656a3476fc.png)

– 49.03 on a reportedly medium screen then a few minutes later, with the same browser session (a heavily extended profile) and the same test in a popped-out maximised window on a 1,920 x 1,080 display: 

![image](https://user-images.githubusercontent.com/192271/69009130-7b5ace80-094a-11ea-939c-b7a87ec3f07c.png)

– 1.0 on a reportedly medium screen.
